src/mumble/Log.cpp

```diff
@@ -571,6 +576,17 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 		}
 	}
 
+	// TTS No Author
+	if (g.s.bTTSNoAuthor){
+		int authorNameLen = 0;
+		authorNameLen = plain.indexOf(QLatin1String(": "), 0) + 2;
+		plain.remove(9, authorNameLen-9);
```

Comment:
does this work in all languages? Without looking, I would say that the colon is not present or the same in all languages.

Reply:
Thanks for the comment! 

I think your right. Do you think it would be better to instead hard coding the offset of 9 I instead simply remove all the chars between the first ")" and ":" that should remove the author from messages like:

```
(tree) username: foo
(channel) username: foo
```

Reply:
My worry is that there could be a translation where there is no parenthesis or colon, in which case you would either get an index of -1 (the prefix would be broken), or a colon in the actual message (the code would eat part of the message). The best way to solve this would be to find the code that creates this formatted string and to create another formatted string there that would be used specifically for TTS. I know that kind of change is a lot more involved, but I don't think manipulating the message already formatted will lead to anything that won't be brittle with regards to internationalisation.

Reply:
I'd imagine that () are always wrapped around the scope and a : always follows the author regardless of what language the user is in. I'll try to audit the code path that gets us to this point to confirm this is the case.

I'd like to solve this issue before having to do string manipulation but I get the impression that the log is actually overloaded, it was intended to be a log of events not a chat box.

Reply:
strings manipulating wont work, these messages are translatable, e.g. zh-CN chat might be:
```
（频道）用户：foo
```
screenshot:
![image](https://user-images.githubusercontent.com/5835759/57219668-63b19d80-702b-11e9-8876-3003922b1d44.png)

both `（）` and `：` are non-latin and there is no whitespace
and i think RTL languages are more complicated